trust/Makefile.am

```diff
@@ -117,6 +117,17 @@ trust_trust_SOURCES = \
 	$(asn_h) \
 	$(NULL)
 
+bin_PROGRAMS += trust/p11-kit-verify
```

Comment:
I have a slight preference to renaming it to `trust verify`, as the format is sorely for trust information.

Reply:
ok, I will change it.

Reply:
After a second thought, `verify` might not be a good choice, as it may imply certificate verification. Maybe `trust check-format` (or similar) might be better?

Reply:
I was unsure of the name as well. trust-check-format sounds more appropriate